packages/devtools-modules/client/shared/components/splitter/SplitBox.js

```diff
@@ -60,6 +60,12 @@ const SplitBox = React.createClass({
     };
   },
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.vert !== nextProps.vert) {
+      this.setState({vert: nextProps.vert});
+    }
```

Comment:
so... should I add them (width/height) or not in your opinion? Sorry, I can't figure out what you are suggesting here 😊